.goreleaser.multi.yml

```diff
@@ -117,3 +117,27 @@ docker_manifests:
 announce:
   discord:
     enabled: true
+
+brews:
+  - name: flipt
+    homepage: https://flipt.io
+    description: An open source, self-hosted feature flag solution
+    license: GPL-3.0-only
+    folder: Formula
+    skip_upload: auto
+
+    post_install: |
+      (var/"log/flipt").mkpath
+
+    service: |
+      run [opt_bin/"flipt"]
+      environment_variables PATH: std_service_path_env
+      keep_alive true
+      error_log_path "#{var}/log/flipt/server-stderr.log"
+      log_path "#{var}/log/flipt/server-stdout.log"
```

Comment:
I was looking at this going... what fresh hell of a configuration language is this.
Then I finally saw it. It's ruby, of course it is... Homebrew.

Reply:
yamlruby

Reply:
I was particularly perturbed by what the hell `opt_bin/` and `var/"flipt"` was.

Then I saw it and a quick GH search turned up the madness:
https://github.com/Homebrew/brew/blob/3c8b4949baefb1f8166749ff1a3f0665afadedda/Library/Homebrew/install_renamed.rb#L29-L31

Reply:
yah a lot of it I copied from Pyroscope's brew formula: https://github.com/pyroscope-io/homebrew-brew/blob/main/Formula/pyroscope.rb#L44